gradle.properties

```diff
@@ -38,112 +43,72 @@ splitties.version=3.0.0-dev-038
 plugin.com.github.ben-manes.versions=0.25.0
 plugin.org.gradle.kotlin.kotlin-dsl=1.2.9
 #                       # available=1.3.1
-plugin.de.fayard.buildSrcVersions=0.6.3
+plugin.de.fayard.buildSrcVersions=0.6.4
```

Comment:
Isn't that a duplicate version number with the one defined in root project's `build.gradle.kts`?